apps/survey/app/src/demo-state.js

```diff
@@ -10,7 +10,7 @@ const historyPoints = prevPoints => {
   const newPoints = prevPoints.map((point, i) => {
     const pointsLeft = prevPoints.length - i
     const maxDiff = remaining / pointsLeft
-    const diff = -maxDiff + maxDiff * Math.random() * 2 / 2
+    const diff = -maxDiff + (maxDiff * Math.random() * 2) / 2
```

Comment:
What's the point of multiply and divide by 2?

Reply:
I guess one of them used to not be a 2 😆, thanks!